lib/relay/relay_pool.dart

```diff
@@ -297,6 +289,15 @@ class RelayPool {
               relay.send(message);
             }
             relay.pendingAuthedMessages.clear();
+
+            // Resubscribe to all active subscriptions after authentication
+            if (relay.hasSubscription()) {
+              var subs = relay.getSubscriptions();
```

Comment:
Use `final` when you're not going to mutate the value, like `let` in Swift.
```suggestion
              final subs = relay.getSubscriptions();
```